### DISCUSSION TITLE
Why Bazelisk version: development ? Not Bazelisk version: v1.20.0 ?

### DISCUSSION THREAD

**Discussion by farcop** [wolfi-dev/os#21663] [General]
I am installing bazelisk-1.20.0-r1 and then run `bazel version` instead of expected `Bazelisk version: v1.20.0` I've got `Bazelisk version: development`

**Comment by MindTooth** [comment]
I've created a [pull request](https://github.com/wolfi-dev/os/pull/21751) in an attempt to fix this.

**Reply by farcop** [reply]
@MindTooth 
Thank you!
